### PR TITLE
Elastic Vector Search new quickstart.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Ticket:**  https://issues.newrelic.com/browse/NR-164848
**Description:**  Created Elastic Vector Search new quickstart.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/elastic-vector-search/config.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+ 
+  ### Track your app:  
+  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into your app's performance and identify areas of improvement.
+
+
+  ### What’s included in this quickstart?
+
+  New Relic LangChain monitoring quickstart provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your LangChain app. These reports include:
```
Comment (rahulbasu-nr) [inline quickstarts/elastic-vector-search/config.yml:17]:
```suggestion
  Elastic Vector Search uses New Relic Langchain monitoring which provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your Elastic Vector Search app. These reports include:
```

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Make these changes.

File: quickstarts/elastic-vector-search/config.yml (hunk 1 above)
Comment (pkudikyala) [reply quickstarts/elastic-vector-search/config.yml]: Hi @rahulbasu-nr ,
Updated the review changes. Kindly review it.

Thanks...

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Review by pkosanam — APPROVED** [review]
lgtm

**Pull request merged**